lookout/style/typos_checker/README.md

```diff
@@ -1,53 +1,86 @@
 # Typos checker
 
-Steps to reproduce the demo as it was seen during the project presentation.
+**Typos checker** is a module for **Style Analyzer** for **[lookout](https://github.com/src-d/lookout)**, which points out typos in code comments when they're introduced or modified by a Pull Request.
 
-PR and issues are welcomed ;)
+# How to run it
 
-## how to reproduce
+You can follow these steps to reproduce the demo, as it was seen during the presentation of the hackathon project.
+
+PR and issues are very welcomed ;)
+
+## Run lookout
+
+You will need to have [Go](https://golang.org/doc/install#install) and [docker-compose](https://docs.docker.com/compose/install/)
```

Comment:
suggestion: You will need to have [Go](https://golang.org/doc/install#install) and [docker-compose](https://docs.docker.com/compose/install/) installed in your machine